core/src/it/resources/reference.conf

```diff
@@ -0,0 +1,18 @@
+akka {
+  loglevel = "DEBUG"
+
+  actor {
+    serialize-messages = on
```

Comment:
Why?

Reply:
Copy-pasta from the reference.conf from tests.